repository/Seaside-Tools-Web.package/WAFaviconHandler.class/instance/handleFiltered..st

```diff
@@ -0,0 +1,7 @@
+handling
+handleFiltered: aRequestContext
+
+	aRequestContext response
+		contentType: (WAMimeType main: 'image' sub: 'x-icon');
```

Comment:
While not necessary, it might make sense to add a convenience method for this mimetype along the way.